schemas/aws/account-1.yml

```diff
@@ -167,6 +170,11 @@ dependencies:
         - false
     required:
     - sso
+  organizationAccountTags:
+    properties:
+      organization:
+        enum:
+        - null
```

Comment:
why `organizationAccountTags` requires `organization` to be `null`?

Reply:
If an account has an `organization` set, it's an organization account; if not, it's a payer account. Only payer accounts can have the `organizationAccountTags` attribute.

Reply:
can we add this limit to `description`? like only available for payer account.

Reply:
Sure. Done in [6b2d9c6](https://github.com/app-sre/qontract-schemas/pull/785/commits/6b2d9c6575da13645e46ba6240b78b06320d2158)